### ISSUE TITLE
fr2en - How were the bin files created?

### ISSUE THREAD

**Issue by mattiasmar** [pytorch/glow#2841]
In the example [fr2en](https://github.com/pytorch/glow/blob/master/examples/fr2en.cpp) there is use of some bin and txt files. How were they created? 

Thanks!

**Comment by jfix71** [comment]
CC @artemrakhov

**Comment by mattiasmar** [comment]
@jfix71  @artemrakhov  ping!

**Comment by artemrakhov** [comment]
Sorry for the delay. I followed this tutorial: 
https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html
I took code from there and partially rewrote some parts. Removed dropout and attention, in order to keep the model as simple as possible (but still correct). I trained the model locally in PyTorch for less than 1 hour. Then I dumped all embedding tables, weights and biases to bin files by raw Python means.
Txt files are simply the lists of used words for both languages (French and English), also generated by the code from tutorial. Note that these lists are in random order, i.e. these are not dictionaries and don't match by meanings. The lists are based on the most common words from following English to French translation pairs:
https://www.manythings.org/anki/fra-eng.zip
